src/burnchains/mod.rs

```diff
@@ -521,6 +521,7 @@ pub struct BurnchainView {
     pub burn_stable_block_height: u64, // latest stable block height (e.g. chain tip minus 7)
     pub burn_stable_block_hash: BurnchainHeaderHash, // latest stable burn block hash
     pub last_burn_block_hashes: HashMap<u64, BurnchainHeaderHash>, // map all block heights from burn_block_height back to the oldest one we'll take for considering the peer a neighbor
+    pub rc_consensus_hash: ConsensusHash, // consensus hash of the current reward cycle's start block
```

Comment:
This comment should be a rustdoc instead.

Reply:
Fixed